setup/upgrade_minor.rst

```diff
@@ -21,9 +21,10 @@ There are two steps to upgrading a minor version:
 1) Update the Symfony Library via Composer
 ------------------------------------------
 
-Your ``composer.json`` file should already be configured to allow your Symfony
-packages to be upgraded to minor versions. But, if a package was not upgraded,
-check that the version constrains of your Symfony dependencies are like this:
+Your ``composer.json`` file is configured to allow your Symfony
```

Comment:
Wouldn't it be better to use `the` instead of `you`?

```suggestion
The ``composer.json`` file is configured to allow your Symfony
```

In full:

```suggestion
The ``composer.json`` file is configured to allow Symfony
```